scripts/prep_data_for_model.R

```diff
@@ -106,12 +107,16 @@ gcs_upload(standard_release,
            name = "jpe-model-data/release_summary.csv",
            predefinedAcl = "bucketLevel")
 write_csv(standard_release, "data/model-data/release_summary.csv")
-gcs_upload(standard_recapture,
+# add zero recaptures
```

Comment:
This all makes sense. Is there any reason that we would also want to add this to the standard_recaps table? 

I don't think it is needed but worth thinking about